### PR TITLE
Add error info when release component clone with inactive

### PR DESCRIPTION
**Pull request by bliuredhat**
Info error when clone release component for target release
which with inactive.

JIRA: PDC-1282

### CODE REVIEW

File: pdc/apps/release/views.py [hunk 1]
```diff
@@ -694,6 +694,10 @@ def create(self, request):
         if source_release.releasecomponent_set.count() == 0:
             return Response({'detail': 'there is no component in source release'},
                             status=status.HTTP_204_NO_CONTENT)
+        if not target_release.active:
+            return Response({'detail': 'can\'t clone components' +
+                                       'for target release %s with inactive' % target_release_id},
```
Comment (simozhan) [inline pdc/apps/release/views.py]: Is it better to change the detail to "can't clone components to an inactive target release %s"

File: pdc/apps/release/views.py [hunk 2]
```diff
@@ -694,6 +694,10 @@ def create(self, request):
...
+                            status=status.HTTP_406_NOT_ACCEPTABLE)
```
Comment (simozhan) [inline pdc/apps/release/views.py]: It seems HTTP_406_NOT_ACCEPTABLE is according to Accept Headers. https://en.wikipedia.org/wiki/List_of_HTTP_status_codes
I think HTTP_400_BAD_REQUEST should be ok.

File: pdc/apps/release/views.py (hunk 1 above)
Comment (bliuredhat) [inline pdc/apps/release/views.py]: @simozhan ; your ideal is more clear~,thank you~!

**Comment by simozhan** [comment]
:+1:

**Comment by ycheng-aa** [comment]
:+1:

**Pull request merged**